code/modules/jobs/job_types/engineering.dm

```diff
@@ -14,6 +14,9 @@ Chief Engineer
 	selection_color = "#ffeeaa"
 	req_admin_notify = 1
 	minimal_player_age = 7
+	exp_requirements = 600

```

Comment:
There already is a config for overriding the time needed for head of staff jobs.
Also, tracking time, using the restriction for head of staff jobs, using it for non head of staff jobs, and requiring department specific playtime are all separate configs.